target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-ldf-5-ac.dts

```diff
@@ -0,0 +1,251 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/* Copyright (c) 2020 Robert Marko <robimarko@gmail.com> */
```

Comment:
Why somebody else Copyright here?

Reply:
Matter of courtesy and credits. Feels odd to put my name when > 95% is the work of others. But if it is mandatory I can put my ID there.

Reply:
Well, you never told that 95 % is the work of others, so how should I now that?